frontend/src/components/common/Terminal.tsx

```diff
@@ -162,9 +163,9 @@ export default function Terminal(props: TerminalProps) {
     // The first byte is discarded because it just identifies whether
     // this data is from stderr, stdout, or stdin.
     const data = bytes.slice(1);
-    const text = decoder.decode(data);
+    let text = decoder.decode(data);
 
-    if (bytes.byteLength < 2) {
+    if (bytes.byteLength < 1) {
```

Comment:
Is this needed in the end if the first byte is just telling which channel to use?